src/priors/ltisde.jl

```diff
@@ -21,9 +21,23 @@ end
 drift(sde::LTISDE) = sde.F
 dispersion(sde::LTISDE) = sde.L
 
+iterate(sde::LTISDE) = sde.F, true
+iterate(sde::LTISDE, s) = s ? (sde.L, false) : nothing
+length(sde::LTISDE) = 2
+
 discretize(sde::LTISDE, dt::Real) =
     matrix_fraction_decomposition(drift(sde), dispersion(sde), dt)
```

Comment:
Then maybe add a test to check that matrix_fraction_decomposition and FHG do similar things